Gems/ROS2/Code/Source/Lidar/LidarSensorConfiguration.h

```diff
@@ -39,13 +40,20 @@ namespace ROS2
         AZStd::unordered_set<AZ::u32> m_ignoredCollisionLayers;
         AZStd::vector<AZ::EntityId> m_excludedEntities;
 
+        AZStd::vector<AZStd::tuple<AZStd::string, uint8_t, AZ::Color> > m_segmentationClasses = {
+            {"unknown", 0, AZ::Color(1.0f, 1.0f, 1.0f, 1.0f)}
+        };
+
+        [[nodiscard]] AZStd::array<AZ::Color, 256> GenerateSegmentationColorsLookupTable() const;
```

Comment:
256 here is probably uint8_t size for segment id - I would think for a while, how sizeof segment id can be placed here instead.